### PR TITLE
fix setup.py chicken and egg version.py problem.

### PR DESCRIPTION
**Pull request by CyclingNinja**
Needed to move the import EISpy to after the generate_version_py call.

### CODE REVIEW

**Comment by dpshelio** [comment]
Is this a problem from the template of affiliated packages?
@Cadair, you know this better than me...

File: eispy/calibration/data_calibration.py [hunk 1]
```diff
@@ -136,12 +136,15 @@ def _get_pixel_solid_angle(detector, slit):
     omega-d in the EIS software note 02.
     """
     mirror_focus_length = 1938.68 * u.mm
+    print(slit)
     if slit == 1:
```
Comment (CyclingNinja) [inline eispy/calibration/data_calibration.py]: For some reason my data doesn't have a slit number, so we added this fix

File: eispy/calibration/data_calibration.py [hunk 2]
```diff
@@ -112,9 +112,9 @@ def radiometric_calibration(meta, *data_and_errors, **kwargs):
         seconds_per_exposure = total_time / data.shape[0]
         err = _calculate_errors(data, err, index, meta)
         data /= seconds_per_exposure
```
Comment (CyclingNinja) [inline eispy/calibration/data_calibration.py]: Fixing quantity issues

File: setup.py [hunk 3]
```diff
@@ -33,10 +33,6 @@
 LICENSE = metadata.get('license', 'unknown')
 URL = metadata.get('url', 'http://astropy.org')
```
Comment (CyclingNinja) [inline setup.py]: Chicken and Egg issue fix

File: eispy/calibration/pixel_calibration.py [hunk 4]
```diff
@@ -224,10 +224,10 @@ def _construct_hot_warm_pix_url(date, pix_type, detector, top_bot, left_right):
     url += pix_type + '/'
     url += date.strftime("%Y-%m-%d") + '/'
     url += 'coords_' + detector + '_' + left_right + '_'
```
Comment (CyclingNinja) [inline eispy/calibration/pixel_calibration.py]: When we ran this and it errored with astropy

File: eispy/calibration/data_calibration.py (hunk 1 above)
Comment (Cadair) [inline eispy/calibration/data_calibration.py]: yeah, the slit number was coming out as zero, so this was a hack, no idea what the purpose of this function is  though.

Comment (dpshelio) [inline eispy/calibration/data_calibration.py]: @CyclingNinja could you tell me an example file without slit number? is it an slot file?

File: eispy/calibration/pixel_calibration.py (hunk 4 above)
Comment (dpshelio) [inline eispy/calibration/pixel_calibration.py]: This is to do with writing the file (level 1) and not being in standard... I'll have to look in detail.

**Comment by dpshelio** [comment]
@CyclingNinja I have forgotten about it - do we need to merge this?

**Comment by wtbarnes** [comment]
I think this PR is probably sufficiently stale and can be closed in light of the latest package template updates?

**Pull request closed without merging**